staubli_resources/urdf/common_materials.xacro

```diff
@@ -4,13 +4,25 @@
 
   <xacro:macro name="material_staubli_yellow">
     <material name="">
-      <color rgba="${colour_staubli_yellow}"/>
+      <color rgba="${melon_yellow}"/>
+    </material>
+  </xacro:macro>
+  
+  <xacro:macro name="material_staubli_yellow_b">
+    <material name="">
+      <color rgba="${zinc_yellow}"/>
     </material>
   </xacro:macro>
 
   <xacro:macro name="material_staubli_white">
     <material name="">
-      <color rgba="${colour_staubli_white}"/>
+      <color rgba="${traffic_white}"/>
+    </material>
+  </xacro:macro>
+	
+  <xacro:macro name="material_traffic_grey_b">
+    <material name="">
+      <color rgba="${traffic_grey_b}"/>
     </material>
   </xacro:macro>
 
```

Comment:
Here we should just mimic the 'new' colour naming scheme, so `material_ral_melon_yellow`, etc.